scripts/verify

```diff
@@ -40,6 +40,10 @@ case $i in
     additional_deployer_role="${i#*=}"
     shift
     ;;
+  --image_pull_secret=*)
+    image_pull_secret="${i#*=}"
```

Comment:
per offline discussion: image_pull_secret_name

Reply:
Per further discussion keeping image_pull_secret to keep consistent with the same parameter in provision.py